<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1002</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>20</y>
      <width>461</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Local start</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>431</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of tickets for every ticket type:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Prc">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>431</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Params">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>431</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>LotteryTicketsNumber 1 50 LotteryTicketsNumber 2 40 LotteryTicketsNumber 3 30</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>431</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Percentage of winning tickets:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Start">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>501</width>
      <height>431</height>
     </rect>
    </property>
    <property name="title">
     <string>Lottery factory</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>481</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Settings file name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_FN">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>481</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>481</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Factory lines:</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>481</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>471</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Full work time will be:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_WorkTime">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0 seconds</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>70</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of ticket types:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_TicketTypesNum">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>90</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>230</y>
      <width>461</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Network params</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_IPAddr">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>431</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>localhost</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Port">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>431</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1234</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>431</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Port:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>IP address:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>431</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Server status:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_SrvStatus">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>431</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1002</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
